go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.32.1
 	github.com/prometheus/exporter-toolkit v0.7.1
+	github.com/stretchr/testify v1.4.0
```

Comment:
I don't see changes in `go.sum`. Probably you forgot to make `go mod tidy`

Reply:
nope, I run go mod tidy, and see not changes.
but thanks for noticing that